xcdat/__init__.py

```diff
@@ -22,7 +16,11 @@
     create_zonal_grid,
 )
 from xcdat.spatial import SpatialAccessor  # noqa: F401
-from xcdat.temporal import TemporalAccessor  # noqa: F401
+from xcdat.temporal import (  # noqa: F401
+    TemporalAccessor,
+    contains_cftime_datetimes,
+    get_datetime_like_type,
```

Comment:
I don't mind making these so visible, but I'm curious why the end-user would need/want these? I see them mostly used behind-the-scenes.

Reply:
I initially implemented them as private methods, but I thought it would be useful to have them available for users who wanted to implement some more advanced functionality (which is also why I think Xarray made their version of [`contains_cftime_datetime()`](https://github.com/pydata/xarray/blob/c3224ef599e701efb00bbf4c20f8cf5e247a021d/xarray/core/common.py#L1816-L1818)) public.

I can see an argument for keeping them private because most users of xCDAT are probably focused on the core functionality of temporal averaging, regridding, etc. Unless there is a demand for this kind of feature (there probably won't be), I will revert them back to being private.